runhouse/resources/functions/aws_lambda.py

```diff
@@ -433,32 +448,18 @@ def _rh_wrapper(self):
         f.write("def lambda_handler(event, context):\n")
         f.write(f"\tif not os.path.isdir('{self.HOME_DIR}'):\n")
         f.write(f"\t\tos.mkdir('{self.HOME_DIR}')\n")
+        f.write(
+            f"\tif not os.path.isdir('{self.HOME_DIR}/runhouse'):\n"
+            f"\t\tsubprocess.call(['pip', 'install', 'runhouse', '-t', '{self.HOME_DIR}/'])\n"
+            f"\tsys.path.insert(1, '{self.HOME_DIR}/')\n\n"
+        )
```

Comment:
Separate PR, but I continue to feel all this would be a lot easier to manage if we just put it into a file in servers/ rather than write it out like this. We can talk more about how that would look offline.